ui/src/PrintFulfilment.js

```diff
@@ -173,22 +224,29 @@ class PrintFulfilment extends Component {
                 <InputLabel>Export File Template</InputLabel>
                 <Select
                   onChange={this.onPrintTemplateChange}
-                  value={this.state.packCode}
-                  error={this.state.packCodeValidationError}
+                  value={this.state.selectedTemplate}
+                  error={this.state.templateValidationError}
                 >
                   {fulfilmentPrintTemplateMenuItems}
                 </Select>
               </FormControl>
               <FormControl fullWidth={true}>
                 <TextField
-                  style={{ minWidth: 200 }}
+                  style={{ minWidth: 200, marginBottom: 20 }}
                   error={this.state.printUacQidMetadataValidationError}
                   label="UAC QID Metadata"
                   id="standard-required"
                   onChange={this.onNewActionRulePrintUacQidMetadataChange}
                   value={this.state.newPrintUacQidMetadata}
                 />
               </FormControl>
+              {!(this.state.selectedTemplate === null) &&
```

Comment:
🎨 linter says this could be done as `{(this.state.selectedTemplate !== null) &&` but I'm not sure if that's ickier?

Reply:
👍 I use `!==` on the next line so better for consistency